v2/internal/project/project.go

```diff
@@ -119,6 +137,73 @@ func (p *Project) Save() error {
 	return os.WriteFile(p.filename, data, 0755)
 }
 
+func (p *Project) setDefaults() {
+
+	if p.Path == "" {
+		p.Path = lo.Must(os.Getwd())
+	}
+	if p.Version == "" {
+		p.Version = "2"
+	}
+	// Create default name if not given
+	if p.Name == "" {
+		p.Name = "wailsapp"
+	}
+	if p.AssetDirectory == "" {
+		p.AssetDirectory = "assets"
+	}
+	if p.OutputFilename == "" {
+		p.OutputFilename = p.Name
+	}
+	if p.FrontendDir == "" {
+		p.FrontendDir = "frontend"
+	}
+	if p.WailsJSDir == "" {
+		p.WailsJSDir = p.FrontendDir
+	}
+	if p.DebounceMS == 0 {
+		p.DebounceMS = 100
+	}
+	if p.DevServer == "" {
+		p.DevServer = "localhost:34115"
+	}
+	if p.NSISType == "" {
+		p.NSISType = "portable"
```

Comment:
IIRC portable is not a valid value for NSISType, it should be one of "single"/"multiple" and the default was empty which is mapped to "multiple".

```suggestion
		p.NSISType = "multiple"
```

Reply:
WTF copilot! Oooph. Was this even set before?

Reply:
It was empty, which mapped to multiple.